### PR TITLE
Add UI SyncPlan synchronization tests

### LINKED ISSUE #3276: Augment test coverage for sync plan functionality
It is really nice idea to create sync plans for future period of time (like in 5 minutes from current time) and try to assign that plan to the product. After that it is necessary to verify that product repositories are actually synchronized.

Also, it is necessary to verify that sync plans interval functionality (at least, 'hourly') is really working. That means that we need to check that product is re-synchronized in one hour after first sync is happened

### PR DESCRIPTION
**Pull request by abalakh**
Similar to #3277 and #3274, UI part.
Closes #3276

### CODE REVIEW

**Comment by abalakh** [comment]
Tests are running right now, will add the resultsa bit later.

File: robottelo/ui/locators.py [hunk 1]
```diff
@@ -1383,6 +1383,15 @@ def __iter__(self):
     "repo.fetch_upstream": (
         By.XPATH, ("//form[@bst-edit-text='repository.docker_upstream_name']"
                    "/div[@class='bst-edit']/div/span[2]")),
+    "repo.fetch_packages": (
+        By.XPATH, "//td[span[text()='Packages']]/following-sibling::td",
```
Comment (abalakh) [inline robottelo/ui/locators.py]: Don't like the idea to rely on localized strings, but those fields have no unique ids, no classes, nothing intresting that we could use. The only other option is to rely on indexes which is even worse, as for different types of packages we have different amount of fields.

File: robottelo/ui/products.py [hunk 2]
```diff
@@ -57,18 +57,18 @@ def update(self, name, new_name=None, new_desc=None,
             if new_desc:
                 self.click(locators['prd.desc_edit'])
                 self.text_field_update(locators['prd.desc_update'], new_name)
-                self.click(common_locators['create'])
+                self.click(common_locators['save'])
```
Comment (abalakh) [inline robottelo/ui/products.py]: Just accidentaly found that bug while playing with adding a sync plan to package (tests are using adding package to syncplan however). As it's update func, of course there's no `create` button.

File: robottelo/ui/locators.py (hunk 1 above)
Comment (omaciel) [inline robottelo/ui/locators.py]: How about we fix this upstream as well, by providing a PR that would give us a unique ID? We've done it in the past (ask @oshtaier for more info).

**Comment by abalakh** [comment]
Tests results uncovered one issue with nailgun.
Now this PR depends on SatelliteQE/nailgun#262

File: robottelo/ui/locators.py (hunk 1 above)
Comment (oshtaier) [inline robottelo/ui/locators.py]: For my case, it was really impossible to address proper DOM element without reference to really dynamic things. Here we have good enough locators that will work just nice for our application. Yes, that is not best idea to access element by text, but we have huge amount of locators that work in the same way. Don't think that we need to make current case so unique and spend time on fixing Foreman application itself (create bug, find solution, fix the code, ...).
@omaciel Please, let us know if we need do that anyway

**Comment by oshtaier** [comment]
What is a connection between that UI PR and SatelliteQE/nailgun#262?

**Comment by oshtaier** [comment]
ACK pending questions and test results

**Comment by abalakh** [comment]
Alhough it's UI PR, i'm creating all additional objects like products or repos via API to save some time (as these are sync plan tests and we are not testing product/repo creation here). Direct answer to your question is - lines like [repo.product.read().name](https://github.com/SatelliteQE/robottelo/pull/3314/files#diff-77f1c30edf0513bf42566027dc5216d6R555) won't work. It's not really crucial part of PR, i can rewrite the test a bit to avoid such api calls if we are in hurry, but if we aren't - a little bit better to leave it as it is, as otherwise i'll have to introduce more code and nested loops.

**Comment by abalakh** [comment]
Test results (with SatelliteQE/nailgun#262 applied):
```python
============================= test session starts ==============================
platform linux2 -- Python 2.7.10, pytest-2.8.7, py-1.4.31, pluggy-0.3.1
rootdir: /home/andrii/workspace/robottelo, inifile: 
plugins: xdist-1.13.1, repeat-0.2
collected 19 items

tests/foreman/ui/test_syncplan.py ss..s.

 13 tests deselected by '-ktest_positive_synchronize or test_negative_synchronize' 
============ 3 passed, 3 skipped, 13 deselected in 2463.96 seconds =============
```

**Comment by elyezer** [comment]
ACK, further improvements can be done later.

**Pull request merged**